lib/trento_web/controllers/v1/settings_controller.ex

```diff
@@ -147,6 +147,44 @@ defmodule TrentoWeb.V1.SettingsController do
     end
   end
 
+  operation :get_analytics_settings,
+    summary: "Fetches the Analytics settings",
+    tags: ["Platform"],
+    responses: [
+      ok:
+        {"Analytics settings fetched successfully", "application/json",
+         Schema.Platform.AnalyticsSettings},
+      not_found: Schema.NotFound.response()
+    ]
+
+  def get_analytics_settings(conn, _) do
+    case Settings.get_analytics_settings() do
+      {:ok, settings} -> render(conn, :analytics_settings, %{settings: settings})
+      {:error, :analytics_settings_not_configured} -> {:error, :not_found}
+    end
+  end
+
+  operation :update_analytics_settings,
+    summary: "Updates the Analytics settings",
+    tags: ["Platform"],
+    request_body: {"AnalyticsSettings", "application/json", Schema.Platform.AnalyticsSettings},
```

Comment:
I'm afraid this `request_body` cannot be the same as the response hehe.
You will need to create a new schema like `UpdateAnalyticsSettingsRequest` which will include the `optin` value.
By coincidence they match now, but we need 2 different schemas

Reply:
Cool, I have created an `AnalyticsSettingsUpdateRequest`.